addons/website_sale/views/templates.xml

```diff
@@ -873,7 +873,7 @@
     <template id="extra_info" name="Checkout Extra Info">
         <t t-call="website.layout">
             <t t-set="no_footer" t-value="1"/>
-            <div id="wrap">
+            <div id="wrap" class="o_ws_nofooter">
```

Comment:
It seems there is an oe_structure *after* that wrap so this fix would only work if nothing in that oe_structure